internal/shared/logutil/convert.go.tmpl

```diff
@@ -1,7 +1,10 @@
+// Code created by gotmpl. DO NOT MODIFY.
+// source: internal/shared/logutil/convert.go.tmpl
+
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-package otellogr // import "go.opentelemetry.io/contrib/bridges/otellogr"
+package logutil // import "go.opentelemetry.io/contrib/internal/shared/logutil"
```

Comment:
Can you make the the package name as an template parameter/input and put it directly to the pacakge (instead of inside `internal/logutil`?